python/orca/test/bigdl/orca/learn/ray/experimental/test_mmcv_estimator_ray_backend.py

```diff
@@ -218,6 +283,28 @@ def test_run_with_batch_processor(self):
         print(f"dLoss: {dloss}")
         assert dloss < 0
 
+    def test_get_model(self):
+        estimator = get_estimator(linear_model_runner_creator)
+        estimator.run([simple_dataloader_creator], [('train', 1)])
+
+        # test get_state_dict()
+        state_dict = estimator.get_state_dict()
+        assert 'epoch' in state_dict
+        assert 'model' in state_dict
+        assert 'optimizer' in state_dict
+        self.assertEqual(state_dict['epoch'], 30)
+
+        # test get_model()
+        model_state = estimator.get_model()
+        weight = model_state["fc1.weight"].item()
+        assert abs(weight - 2.0) < 0.0001
```

Comment:
we should:
1. train a model, record its loss or something indicates its performance
2. save the model, and then load it
3. validate its performance